tests/txt.test.js

```diff
@@ -406,14 +407,14 @@ describe('The document has more than 15 pages and not Table of Contents.', () =>
     await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.FORGIVE_CHECKLIST })).resolves.toHaveLength(0)
     await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.SUBMISSION })).resolves.toHaveLength(0)
   })
-  test('Table of Contents missing and pages less 15', async () => {
+  test('Table of Contents missing and pages are current threshold', async () => {
```

Comment:
can we interpolate the PAGE_THRESHOLD_REQUIRING_TOC into this string?
"and there are more than ${PAGE_THRESHOLD_REQUIRING_TOC} pages?" or the like?

Reply:
if not, the string should be "there are more pages than the current threshold".